packages/compass-e2e-tests/helpers/commands/run-find-operation.ts

```diff
@@ -98,7 +101,7 @@ async function maybeResetQuery(browser: CompassBrowser, tabName: string) {
   );
   await resetButton.waitForDisplayed();
 
-  if (!(await resetButton.getAttribute('class')).includes('disabled')) {
+  if (await resetButton.isEnabled()) {
```

Comment:
I am not sure how this is working on main right now, we didn't have disabled class applied to this button for at least half a year now. But this was causing a failure specifically in this branch 🤔